tools/splat_ext/pm_sprites.py

```diff
@@ -832,11 +835,17 @@ def split(self, rom_bytes) -> None:
 
     def get_linker_entries(self):
         from splat.segtypes.linker_entry import LinkerEntry
+        import splat.scripts.split as split
 
         src_paths = [options.opts.asset_path / "sprite"]
 
-        # for NPC
-        src_paths += [options.opts.asset_path / "sprite" / "npc" / sprite_name for sprite_name in self.npc_cfg]
+        # read npc.xml - we can't use self.npc_cfg because nonvanilla asset packs can change it
+        # for each sprite, add to src_paths
+        asset_stack = tuple(Path(p) for p in split.config["asset_stack"])
```

Comment:
In the future, I think we should have a "custom options" mechanism that exposes project-defined options to segments, so we don't have to do stuff like this

Reply:
agreed